dftimewolf/lib/containers/containers.py

```diff
@@ -104,6 +104,36 @@ def __init__(self, path, filter_expression, project_name):
     self.project_name = project_name
 
 
+class AWSLogs(interface.AttributeContainer):
+  """AWS logs container.
+
+  Attributes:
+      path (str): path to a AWS log file.
+      profile_name (str): the profile used to collect logs.
+      query_filter (str): the query filter used in the log query.
+      start_time (str): the start time used in the log query.
+      end_time (str): the end time used in the log query.
```

Comment:
Maybe just specify the format here too?

Reply:
Realised the docstring was lying and I was actually passing/storing datetime objects. __init__ now casts to str and I've updated the docstrings.